Agreements/Internal/IoM_7.txt

```diff
@@ -1 +1,5 @@
-COPX:EIDW:*:ROLEX:*:*:Isle of Man:London S7:28000:*:ROLEX
\ No newline at end of file
+COPX:EIDW:*:ROLEX:*:*:Isle of Man:London S7:28000:*:ROLEX
+COPX:EIDW:*:BAGIT:*:*:Isle of Man:London S7:28000:*:BAGIT
+COPX:EIDW:*:BAGIT:*:*:Isle of Man:London S7:28000:*:DONAX
+COPX:EIDW:*:BAGIT:*:*:Isle of Man:London S7:28000:*:ALAVA
```

Comment:
Correct the fixes so the agreement gets activated

Reply:
Oops!